prow/plugins/golint/suggestion/golint_suggestion_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package suggestion
+
+import (
+	"go/token"
+	"testing"
+
+	"github.com/golang/lint"
+)
+
+func TestLintNamesUnderscore(t *testing.T) {
+	var testcases = []struct {
+		problem  lint.Problem
+		expected string
+	}{
+		{
+			lint.Problem{
```

Comment:
Please specify the field names for the test case struct literals in this file as well.